docs/source/conf.py

```diff
@@ -28,7 +28,7 @@
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #
-# needs_sphinx = '1.0'
```

Comment:
Bikeshed: Do we need to keep this line in the file? Isn't having the old version only in VCS good enough?